governance/xc_admin/packages/xc_admin_frontend/.eslintignore

```diff
@@ -1 +1,3 @@
 node_modules
+tailwind.config.js
```

Comment:
Just curious about what made you add these here, seems like they're already ignored by default?

Reply:
When I ran the eslint command from inside the project folder it included these files. It's good to have them here to.